src/components/icon/CdrIcon.vue

```diff
@@ -41,8 +41,8 @@ export default defineComponent({
     const baseClass = 'cdr-icon';
     const hideSr = !ctx.attrs['aria-label'] && !ctx.attrs['aria-labelledby'];
     const inheritColorClass = computed(() => props.inheritColor
-      && buildClass(baseClass, 'inherit-color'));
-    const sizeClass = computed(() => buildClass('cdr-icon', props.size));
+      && `${baseClass}--inherit-color`);
+    const sizeClass = computed(() => props.size && responsiveModifyClass(baseClass, '', props.size));
```

Comment:
This `responsiveModifyClass` is still needed when a prop can accept breakpoint arguments (e.g. `small@lg`)